portality/templates/application_form/_application_diff.html

```diff
@@ -21,6 +21,9 @@ <h2 class="alert">
             {% set new = form_diff[f.name]['b'] %}
             <tr>
                 <td>
+                    {% if f.diff_table_context %}
+                        <p class="form__hint">{{ f.diff_table_context }} information</p>
```

Comment:
I think it could also look good / clear using the `label` class, wanna try it with that?